third_party/terraform/resources/resource_google_organization_iam_custom_role.go

```diff
@@ -122,6 +122,7 @@ func resourceGoogleOrganizationIamCustomRoleRead(d *schema.ResourceData, meta in
 	d.Set("role_id", parsedRoleName.Name)
 	d.Set("org_id", parsedRoleName.OrgId)
 	d.Set("title", role.Title)
+	d.Set("name", role.Name)
```

Comment:
`name` won't appear in state until the field is established as part of the schema. It will need to be set to `Computed: true` as well.

Reply:
Do I set this in the schema definition on resourceGoogleProjectIamCustomRole function?

Reply:
Yep, within the schema map. Just needs to be set to a string and computed.